### DISCUSSION TITLE
Cambio porta HA

### DISCUSSION THREAD

**Discussion by magomer** [indomus/forum#1829] [HUB: Home Assistant]
Ciao a tutti. 
Vorrei cambiare porta ad una installazione di home assistant os su mini pc.
Utilizzarne un'altra al posto della 8123.
Suggerimenti ?.
Grazie 

Marco

**Comment by snakuzzo** [comment]
Guarda la doc ufficiale [QUI](https://www.home-assistant.io/integrations/http/#server_port)

**Reply by magomer** [reply]
Cavolo, non era difficile....
Grazie !!!!

**Reply by snakuzzo** [reply]
Primo risultato su Google ;)

**Reply by magomer** [reply]
Non il mio di Google.....
Con cambio porta Home assistant mi ha rimandato a varie discussioni tra cui una tua.
Certo una letta alla doc ufficiale potevo darla meglio.....
Grazie ancora !

**Reply by snakuzzo** [reply]
Beh ... Per queste cose devi dimenticarti dell'esistenza dell'italiano. Queste ricerche si fanno in lingua inglese e bastano anche solo le parole chiave

**Reply by magomer** [reply]
E anche su questo hai perfettamente ragione.
Fatta ricerca in inglese e....è spuntata la risposta.....
